### DISCUSSION TITLE
Does higher bitrate matters?

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#499] [General]
Hi,

Before someone answers that "GoPro has a tiny sensor and please don't expect much from it", I would like to ask a question regarding bitrate.

In terms of editing (I am using LOGB and applying Leeming LUT), 190 Mbps works great for 5.3K 30fps but when using 5.3K 60fps, it has to be reduced to 170 Mbps.

In terms of quality from the tiny GoPro 11 sensor, is 190 Mbps differ much from 170 Mbps?

:-)

**Comment by dnewman-gpsw** [comment]
Compression is weird math, with increasing resolution it gets more efficient, same with increasing frame rate. This is the case if you sharing the same FOV and FPS as your capture. However, if you crop 8:7 to 16:9 or zoom in you are only sharing a sub region.  So this is why a bump on compression can help.  e.g. if you cropped 4K 16:9 from a 8:7 5.3K frame, you are using only 33.6% of the data (pixels and bits.)  Sharing 4kp30 at 40Mb/s HEVC is excellent (Netflix is 15Mb/s for 4K, for a not completely fare comparison.)  33.6% of 190Mb/s is 64Mb/s so very very good, even overkill.  The same goes for high frame rates, 60fps does not need more bitrate when presented at 60fps, as the frames update faster, compression issues are less visible, so the main issue of concern is slow motion. If you shoot 5.3kp60 16:9 (highest res for 10-bit), but share slow motion crop at 4Kp30, your effective bits are 26.1% (per second, using fuzzy math here.)  So 26.1% of 170Mb/s is 44Mb/s, still excellent. 

Pretty much 150Mb/s is all you need to never have a compression concern for the most aggressive crop and time remap.

**Reply by tetrahydra** [reply]
Interesting answer.

I mostly record in 5.3K30p 16:9

For videos taken on my tasks (as part of the teaching and also during events), they are submitted for archival. Of course, there were no requirements to have a specific bitrate, etc., but I definitely like to send them good-quality ones.

I don't normally shoot for public sharing, but only among students/teachers who are permitted to access our unit's data storage. Some are just personal trips that I only share with family members within the same house, which can be played on our massive TV screen.

Back to the reason why I started this discussion is to find the balance to allow *editing* which is the reason why I am using LOGB and a high bitrate. Unlike editing a 4K video coming out from Sony A7SIII or A7IV which I tend to record at 140Mbps 4:2:2 10-bit which can retain the colour and the sharpness, when exported in FCPX, the output bitrate is set to 80Mbps. I would like to find a balance for my current GoPro 11 where I do not need to edit aggressively when sharpening or removing noises. Just this morning I accidentally opened a clip shot with GoPro 8 from 2021 and I definitely do not miss those days anymore, when I am just satisfied with whatever coming out from my unit. GoPro 11 has definitely come a long way now.

Besides, these are going to be the settings that I am passing on to my students. Therefore, I would definitely like to do a bit more research as "students" tend to stick to the information should they learned about it for the first time.

I guess, I'll update my notes regarding bitrate to 150Mbps for both 30p and 60p.

PS: I do hope that GoPro can release a **Lens Mod** to physically alter the light and let the camera to use a larger sensor area to record in "linear" mode instead of using a software to fix the lens distortion.
